e2e/tests/caseManagementOrder_test.js

```diff
@@ -3,25 +3,69 @@ const standardDirectionOrder = require('../fixtures/standardDirectionOrder.json'
 const dateFormat = require('dateformat');
 
 let caseId;
+let today;
 
 Feature('Case Management Order Journey');
 
 BeforeSuite(async (I) => {
   caseId = await I.submitNewCaseWithData(standardDirectionOrder);
+  today = new Date();
 });
 
 Scenario('local authority sends agreed CMO to judge', async (I, caseViewPage, uploadCaseManagementOrderEventPage) => {
   await I.navigateToCaseDetailsAs(config.swanseaLocalAuthorityUserOne, caseId);
+  await localAuthoritySendsAgreedCmo(I, caseViewPage, uploadCaseManagementOrderEventPage);
+  I.seeEventSubmissionConfirmation(config.applicationActions.uploadCMO);
+  caseViewPage.selectTab(caseViewPage.tabs.draftOrders);
+  assertDraftCaseManagementOrder(I);
+});
+
+Scenario('Judge sends agreed CMO back to the local authority', async (I, caseViewPage, reviewAgreedCaseManagementOrderEventPage) => {
+  await I.navigateToCaseDetailsAs(config.judicaryUser, caseId);
```

Comment:
if there is link in tab, then better scenario would be that judge opens drafts tab, verify drafts are there and click link